x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/expression/function/scalar/string/CIDRMatchFunctionProcessorTests.java

```diff
@@ -36,8 +36,8 @@ public void testCIDRMatchFunctionInvalidInput() {
         ArrayList<Expression> addresses = new ArrayList<>();
 
         // Invalid source address
-        EqlIllegalArgumentException e = expectThrows(
-            EqlIllegalArgumentException.class,
+        Exception e = expectThrows(
```

Comment:
nit: for the updated test cases in this file, I think throwing and expecting a verification exception would be more precise but this is fine as well.